lib/cdo/shared_constants/pd/teacher1920_application_constants.rb

```diff
@@ -129,6 +129,86 @@ def self.clean_multiline(string)
       }
     }.freeze
 
+    CSV_LABEL_OVERRIDES = {
```

Comment:
A lot of these strings are identical to the strings above. These aren't overriding the strings, they are sometimes duplicating them.

I suggest flattening the above hash map, and keeping this as just the strings that differ from the above map. Can explain in more detail in person